### DISCUSSION TITLE
Problem OAuth Discord and Google

### DISCUSSION THREAD

**Discussion by LostPandora** [requarks/wiki#4624] [Error / Bug Report]
Problem oAuth:

I have tried to implement a login with Google and Discord OAuth. In itself it works, but the user who tries to log in does not enter the wiki but ends up on the page where it has no access. Furthermore, the created user does not know the password or even the mail address that led to the creation of the account. Also no confirmation / activation mail is sent. Data in the Oauth match 100% and was also tested with other systems.

**Comment by daBONDi** [comment]
Got the same Problem

Auth is fully working, the user gets created and has correct permission
The Login Response from /login in /modules/users.js is successfully returning the json to the SPA with the JWT.

The oAuth Redirects to Login Page and the it redirects to "/" and here I get a 403 response

- Looks something like the jwt token is invalid?
- Maybe think it is an issue in the SPA?

You find something out?

**Reply by ieaster1** [reply]
If you use the browser dev tools, do you get any useful information on what may be happening?

**Reply by ozdeadmeat** [reply]
How do i do that? What am I looking for?
